esp-hal/src/i2c/master/support/eh.rs

```diff
@@ -0,0 +1,43 @@
+use crate::{
```

Comment:
I think a whole module of eh files might be a little overkill, I think a single eh.rs with internal `mod`'s for each is probably a bit more maintainable. Just my opinion though and I'm happy to concede if the consensus is separate files are better.

Reply:
I could merge eha and eh, but I'd prefer keeping 02 separate. As embedded-hal is stable, I don't expect these interfaces to change, I only expect us to drop eh02.

Reply:
I'm happy with that trade off